site-scrapers/LowellGeneral/index.js

```diff
@@ -205,6 +234,14 @@ async function ScrapeWebsiteData(browser) {
             availability[dateKey].numberAvailableAppointments +=
                 stationAvailability[dateKey];
         }
+
+        // Sort the keys since we get dates out of order.
+        availability = Object.keys(availability)
+            .sort()
+            .reduce((obj, key) => {
+                obj[key] = availability[key];
+                return obj;
+            }, {});
```

Comment:
For what it's worth, lexicographical sorting is not correct for "m/dd/yyyy" dates. 
It won't be a real issue until October, I guess, but October sorts ahead of September.

```
> [ '1/1/2021' ,'9/1/2021', '10/1/2021' ].sort()
[ '1/1/2021', '10/1/2021', '9/1/2021' ]
```

Nothing appears to be zero-filling the months that I can tell (if I'm wrong, sorry!)

(I also dislike the .keys.sort.reduce idiom, but I don't have anything better to offer. ]

Reply:
Good catch, you could use moment.diff as a custom sorter. Something like.
```
Object.keys(availability).sort((a, b) => {
  return moment(a).diff(b);
});
```

Reply:
Good call. I meant to put a lead zero on there like I do the day of month but forgot. I’ll put that in a new PR. Maybe we should validate the date key format?